docs/rest-apis/devportal/app-key-mapping.md

```
@@ -0,0 +1,237 @@
+
+</aside>
+
+<h3 id="create-application-key-mapping-parameters">Parameters</h3>
+
+|Name|In|Type|Required|Description|
+|---|---|---|---|---|
+|body|body|[AppKeyMappingRequest](schemas.md#schemaappkeymappingrequest)|true|Application key mapping payload. Use internal, resident, or STS key managers to generate OAuth keys. For external key managers, provide `clientID` and `tokenDetails.keyType` to map an existing client.|
+|orgId|path|string|true|none|
+
+> Example responses
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)